src/renderer/components/GlobalPage/__tests__/WelcomeView.spec.js

```diff
@@ -0,0 +1,70 @@
+import { render, screen } from 'testingUtility';
+import userEvent from '@testing-library/user-event';
+import { WelcomeView } from '../WelcomeView';
+import * as strings from '../../../../strings';
+import * as constants from '../../../../constants';
+
+describe('/renderer/components/GlobalPage/WelcomeView', () => {
+  let user;
+
+  beforeEach(() => {
+    user = userEvent.setup();
+  });
+
+  it('renders an Welcome page', () => {
+    const {
+      container: { firstChild },
+    } = render(<WelcomeView openAddCloud={() => {}} />);
+
+    const removeHtmlTags = (string) => {
+      return string.replace(/<[^>]*>/g, '');
+    };
+
+    const containerCloudIcon = firstChild.querySelector('svg');
+    const containerCloudIconSize = '80';
+    const containerCloudIconColor = 'var(--textColorAccent)';
+
+    const welcomeTitle = firstChild.querySelector('h1');
+
+    const welcomeDescription = firstChild.querySelector('p');
+
+    const welcomeList = firstChild.querySelector('ul');
+
+    const welcomeLink = firstChild.querySelector('a');
+
+    const welcomeButton = firstChild.querySelector('button');
+
+    expect(firstChild).toBeInTheDocument();
+
+    expect(containerCloudIcon).toHaveAttribute('width', containerCloudIconSize);
+    expect(containerCloudIcon).toHaveAttribute(
+      'height',
+      containerCloudIconSize
+    );
+    expect(containerCloudIcon).toHaveStyle(`fill: ${containerCloudIconColor}`);
+
+    expect(welcomeTitle).toHaveTextContent(
+      removeHtmlTags(strings.welcome.titleHtml())
+    );
+
+    expect(welcomeDescription).toHaveTextContent(strings.welcome.description());
+
+    expect(welcomeList).toHaveTextContent(strings.welcome.listItems().join(''));
+
+    expect(welcomeLink).toHaveAttribute('target', '_blank');
+    expect(welcomeLink).toHaveAttribute('href', constants.repository.readmeUrl);
+    expect(welcomeLink).toHaveTextContent(strings.welcome.link.label());
+
+    expect(welcomeButton).toHaveTextContent(strings.welcome.button.label());
+  });
+
+  it('clicking on button triggers onClick handler', async () => {
```

Comment:
This may seem like splitting hairs (i.e. being too particular), but I'm just trying to help you look at tests with the right perspective, so I hope this help.

"clicking on button triggers onClick handler" isn't really relevant to the WelcomePage component. What you want to test is, "clicking on button triggers openAddCloud handler" -- you don't really know (i.e. this component doesn't really know) __how__ that handler is being triggered (could be a click, could be mouse down, could be hover, could be some other event); you just care that it's __being__ triggered.

It's a subtle difference, but it stresses the principal of "separation of concerns", which will serve you extremely well in any code you write. It applies to nearly all things we do as software engineers when we write code or design systems. 😄 

```suggestion
  it('clicking on button triggers openAddCloud handler', async () => {
```

Reply:
Understand, thanks